lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -72,7 +74,7 @@ def test_valid_launch(self, render):
         Verifies that the LTI launch succeeds when passed a valid request.
         """
         request = build_launch_request()
-        views.lti_launch(request, COURSE_PARAMS['course_id'], COURSE_PARAMS['usage_id'])
+        views.lti_launch(request, str(COURSE_PARAMS['course_key']), str(COURSE_PARAMS['usage_key']))
```

Comment:
Why have the `COURSE_PARAMS` at this point? Why not just directly use the globals COURSE_KEY and USAGE_KEY?